doc/news/changes/minor/20230518Heinz

```diff
@@ -0,0 +1,11 @@
+New: The new function distributed_compute_intersection_locations() computes
```

Comment:
I think it is fair to move this file to major.

Reply:
[85776df](https://github.com/dealii/dealii/pull/15165/commits/85776df0bd86d3e7c9d4d12bc23733d7b6c6ebbb)